examples/research_projects/codeparrot/scripts/test_deduplicate.py

```diff
@@ -0,0 +1,30 @@
+from unittest import TestCase
```

Comment:
This should go into a folder `scripts/tests/`. Could you also add the command to run the test to the README?

Reply:
yes